ompy/matrix.py

```diff
@@ -324,12 +325,13 @@ def plot(self, *, ax: Any = None,
         self.to_mid_bin()
 
         # Set entries of 0 to white
-        current_cmap = cm.get_cmap()
+        current_cmap = copy.copy(cmap if cmap is not None else cm.get_cmap())
         current_cmap.set_bad(color='white')
         mask = np.isnan(self.values) | (self.values == 0)
         masked = np.ma.array(self.values, mask=mask)
 
-        lines = ax.pcolormesh(Eg, Ex, masked, norm=norm, **kwargs)
```

Comment:
I would have assumed that `cmap` can be passed via `kwargs` by the user already

Reply:
Yes, but the issue is that we modify the default global cmap to get zero to be white. That is deprecated.

Reply:
Sorry, yes, of course